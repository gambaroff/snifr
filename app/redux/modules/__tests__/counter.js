import reducer, { INITIAL_STATE, increment, decrement } from '../counter';

test('increment', () => {
  expect(reducer(INITIAL_STATE, increment())).toMatchSnapshot();
});

test('decrement', () => {
  expect(reducer(INITIAL_STATE, decrement())).toMatchSnapshot();
});
