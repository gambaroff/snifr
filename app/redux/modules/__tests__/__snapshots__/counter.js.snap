// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`decrement 1`] = `
Object {
  "value": -1,
}
`;

exports[`increment 1`] = `
Object {
  "value": 1,
}
`;
